### PR TITLE
docs : fix grammar issues, improve wording, add a description for juicefs_vs_alluxio.md

### PR DESCRIPTION
**Pull request by CaitinChen**
No need for Chinese doc updates

### CODE REVIEW

File: docs/en/introduction/comparison/juicefs_vs_alluxio.md [hunk 1]
```diff
@@ -29,48 +30,48 @@ The following table shows the differences of main features between Alluxio and J
 
 ## Storage format
 
-A single file is [stored](../architecture.md#how-juicefs-store-files) in JuiceFS in three levels: chunk, slice and block. A file will be split into multiple blocks, and be compressed and encrypted (optional) before it is stored into object storage.
+In JuiceFS, a single file is [stored](../architecture.md#how-juicefs-store-files) in three levels: chunk, slice, and block. The file is split into multiple blocks and is compressed and encrypted (optional) before it is stored in object storage.
```
Comment (timfeirg) [inline docs/en/introduction/comparison/juicefs_vs_alluxio.md:33]: JuiceFS comes with its own storage format, which splits files into blocks, optionally encrypted and compressed before uploaded to the object storage. Read [how JuiceFS stores files](../architecture.md#how-juicefs-store-files) for more.

File: docs/en/introduction/comparison/juicefs_vs_alluxio.md [hunk 2]
```diff
@@ -29,48 +30,48 @@ The following table shows the differences of main features between Alluxio and J
...
 
-## Zero-effort operation
+## Zero-effort operations
 
-Alluxio's architecture can be divided into 3 components: master, worker and client. A typical cluster consists of a single leading master, standby masters, a job master, standby job masters, workers, and job workers. You need to maintain all these masters and workers by yourself.
+Alluxio's architecture can be divided into three components: master, worker and client. A typical cluster consists of a single leading master, standby masters, a job master, standby job masters, workers, and job workers. You need to maintain all these masters and workers by yourself.
 
-JuiceFS uses Redis or [others](../../guide/how_to_set_up_metadata_engine.md) as the metadata engine. You could easily use the service managed by public cloud provider as JuiceFS's metadata engine without the need for operation.
+JuiceFS uses Redis or [other services](../../guide/how_to_set_up_metadata_engine.md) as the metadata engine. You could easily use the service managed by a public cloud provider as JuiceFS' metadata engine, without any operational overhead.
```
Comment (timfeirg) [inline docs/en/introduction/comparison/juicefs_vs_alluxio.md:77]: more explicit

```suggestion
JuiceFS uses Redis or [other databases](../../guide/how_to_set_up_metadata_engine.md) as the metadata engine. You could easily use the service managed by a public cloud provider as JuiceFS' metadata engine, without any operational overhead.
```

File: docs/en/introduction/comparison/juicefs_vs_alluxio.md [hunk 3]
```diff
@@ -29,48 +30,48 @@ The following table shows the differences of main features between Alluxio and J
 
 ## Storage format
```
Comment (timfeirg) [inline docs/en/introduction/comparison/juicefs_vs_alluxio.md:31]: this section could use more elaboration: sure, they have different storage format, but what does it really mean for end-users? talk more on performance differences on different scenarios cc @xiaogaozi

**Review by timfeirg — APPROVED** [review]

File: docs/en/introduction/comparison/juicefs_vs_alluxio.md (hunk 1 above)
Comment (CaitinChen) [inline docs/en/introduction/comparison/juicefs_vs_alluxio.md:33]:
```suggestion
JuiceFS has its own storage format, where files are divided into blocks, and they can be optionally encrypted and compressed before being uploaded to the object storage. For more details, see [How JuiceFS stores files](../architecture.md#how-juicefs-store-files).
```

Comment (CaitinChen) [reply docs/en/introduction/comparison/juicefs_vs_alluxio.md]: Updated.

**Pull request merged**